### PR TITLE
READY FOR REVIEW - Fix inconsistency between configuration and hooks docs regarding priority setting

### PR DESCRIPTION
**Pull request by posita**
# TL;DR

In `hooks.rst`, the `priority` settings are inverted in the examples (as written, the examples suggest that *lower* numbers are started first). This PR fixes the examples.

# Details

[`configuration.rst`](https://github.com/circus-tent/circus/blob/92aa3c6214e2730a7f67043268a636a5a233a4ce/docs/source/for-ops/configuration.rst) states (*emphasis* added):

> ***priority*** [¶] Integer that defines a priority for the watcher. When the Arbiter do some operations on all watchers, it will sort them with this field, *from the bigger number to the smallest*. Defaults to 0.

But [`hooks.rst`](https://github.com/circus-tent/circus/blob/92aa3c6214e2730a7f67043268a636a5a233a4ce/docs/source/for-devs/writing-hooks.rst) states:

>  With Circus you can order the startup by using the `priority` option: [¶] `...` [¶] With this setup, Circus will start Redis [with `priority = 1`] first and then it will start the queue worker [with `priority = 2`].

I haven't looked at the code, but my observation is that `configuration.rst`is correct and `hooks.rst` has it backwards. This PR fixes that by inverting the priority numbers used in the `hooks.rst` examples.

### CODE REVIEW

**Comment by k4nar** [comment]
Good catch!

Indeed, `configuration.rst` is right: https://github.com/circus-tent/circus/blob/92aa3c6214e2730a7f67043268a636a5a233a4ce/circus/arbiter.py#L473-L474
